src/osx/cocoa/textctrl.mm

```diff
@@ -771,6 +829,12 @@ - (void)controlTextDidEndEditing:(NSNotification *)aNotification
 
     [tv setDelegate: tv];
 
+    m_undoManager = nil;
+    if ( wxPeer && wxPeer->GetParent() && wxPeer->GetParent()->GetPeer() && wxPeer->GetParent()->GetPeer()->GetWXWidget() )
```

Comment:
Pretty minor, but this would read a lot better with some extra temporary variables/nested `if`s.

I also wonder if this condition can actually be false? If it can't, we should use `wxCHECK` here instead of `if`.